estate-services/src/main/resources/application.properties

```diff
@@ -109,3 +109,7 @@ egov.collectionservice.payment.create.path=collection-services/payments/_create
 
 #Receipt
 kafka.topics.receipt.create=egov.collection.payment-create
+
+#Translation Service
+egov.translation.host=http://localhost:8087/
+egov.translation.endpoint.search.path=localization/messages/v1/_search
```

Comment:
Let's rename these as `egov.localization.host` and `egov.localization.endpoint.search.path`